app/src/main/java/ar/com/wolox/android/training/model/User.java

```diff
@@ -0,0 +1,55 @@
+package ar.com.wolox.android.training.model;
+
+
+public class User {
```

Comment:
This object should be Serializable . You can use  @SerializedName("name") on the fields when the name on the Json is something like "personal_phone" , and then private String personalPhone.

It's not necessary to map all the fields either . You can just create the fields that you need.